public/data/en/fields.json

```diff
@@ -421,7 +421,7 @@
     "label": "I have moved since I last registered in this state.",
     "help_text": false,
     "error_msg": false,
-    "instructions": "",
+    "instructions": "Please only select one of the following checkboxes if it is applicable to you.",
```

Comment:
Revert this - it will be overwritten when we pull in all the json from the CMS. I'll tag Susan from content to add this on the CMS side.